src/Files.App/ViewModels/SelectedItemsPropertiesViewModel.cs

```diff
@@ -602,5 +602,23 @@ public bool IsHidden
 			get => isHidden;
 			set => SetProperty(ref isHidden, value);
 		}
+
+		private bool runAsAdmin;
+		public bool RunAsAdmin
```

Comment:
Can you add docs to the properties in this ViewModel?

Reply:
What do you mean?

Reply:
Comments

Reply:
Ok

Reply:
Should I finally do this? (It's not a problem, just confirming after what d2dyno said)

Reply:
This is good as is 👍